google/cloud/aiplatform/training_jobs.py

```diff
@@ -1185,4 +1177,207 @@ def _model_upload_fail_string(self) -> str:
 
 
 class AutoMLTablesTrainingJob(_TrainingJob):
-    pass
+    def __init__(
+        self,
+        display_name: str,
+        optimization_prediction_type: str,
+        optimization_objective: Optional[str],
```

Comment:
All of these optional arguments should default to None.

Reply:
Done